### DISCUSSION TITLE
网站底部浏览量显示不正常

### DISCUSSION THREAD

**Discussion by Ailontania** [hugo-fixit/FixIt#262] [Q&A] [answered]
我刚刚建了网站，但是网站下面的浏览量不正常。不知哪里的问题。
<a href="https://imgse.com/i/pSku0DU"><img src="https://s1.ax1x.com/2023/01/05/pSku0DU.png" alt="pSku0DU.png" border="0" /></a>

**Comment by Lruihao** [comment, marked as answer]
不蒜子浏览量是根据域名来的，所以本地是所有 localhost:1313 的浏览量，部署到自己域名后就行。
